rpc/provider.go

```diff
@@ -0,0 +1,96 @@
+package rpc
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/status-im/status-go/params"
+)
+
+const (
+	ProviderMain                 = "main"
+	ProviderFallback             = "fallback"
+	ProviderStatusProxy          = "status-proxy"
+	ProviderStatusProxyFallback  = ProviderStatusProxy + "-fallback"
+	ProviderStatusProxyFallback2 = ProviderStatusProxy + "-fallback"
```

Comment:
I guess it's a typo, should be`fallback2`

Reply:
You were faster in commenting on that than me updating it. :)
Thanks, updated.